db/models.py

```diff
@@ -50,3 +60,55 @@ class MovieSession(models.Model):
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(
+        settings.AUTH_USER_MODEL, on_delete=models.CASCADE)
+
+    class Meta:
+        ordering = ["-created_at"]
+
+    def __str__(self) -> str:
+        return str(self.created_at)
+
+
+class Ticket(models.Model):
+    movie_session = models.ForeignKey(MovieSession, on_delete=models.CASCADE)
+    order = models.ForeignKey(Order, on_delete=models.CASCADE)
+    row = models.IntegerField()
+    seat = models.IntegerField()
+
+    class Meta:
+        constraints = [
+            UniqueConstraint(fields=["row", "seat", "movie_session"],
+                             name="unique_ticket")
+        ]
+
+    def clean(self) -> Exception:
+        if self.row > self.movie_session.cinema_hall.rows:
+            raise ValidationError(
+                {"row": [
+                    f"row number must be in available range: "
+                    f"(1, rows): "
+                    f"(1, {self.movie_session.cinema_hall.rows})"]})
+
+        if self.seat > self.movie_session.cinema_hall.seats_in_row:
+            raise ValidationError(
+                {"seat": [
+                    f"seat number must be in available range: "
+                    f"(1, seats_in_row): "
+                    f"(1, {self.movie_session.cinema_hall.seats_in_row})"]})
+
+    def __str__(self) -> str:
+        return (f"{self.movie_session.movie} {self.movie_session.show_time} "
+                f"(row: {self.row}, seat: {self.seat})")
+
+    def save(self, *args, **kwargs) -> None:
+        self.full_clean()
+        return super().save(*args, **kwargs)
+
+
+class User(AbstractUser):
+    pass
```

Comment:
The `User` model extends `AbstractUser` but currently does not add any new fields or methods. Ensure that the `AUTH_USER_MODEL` setting in your Django settings is updated to point to this custom user model, even if it's currently empty. Consider whether you need to add custom fields or methods to justify this extension.